src/rgw/rgw_iam_policy.cc

```diff
@@ -117,6 +118,7 @@ static const actpair actpairs[] =
  { "s3:PutObjectTagging", s3PutObjectTagging },
  { "s3:PutObjectVersionTagging", s3PutObjectVersionTagging },
  { "s3:PutReplicationConfiguration", s3PutReplicationConfiguration },
+ { "s3:PutBucketObjectLockConfiguration", s3PutBucketObjectLockConfiguration},
```

Comment:
Is this an actual S3 operation or our own thing? If the latter we should give it a different prefix.

Reply:
It's an actual S3 operation, we can find it in https://docs.aws.amazon.com/AmazonS3/latest/dev/using-with-s3-actions.html